### PR TITLE
Fix OLLAMA_LLM_LIBRARY with wrong map name and add more env vars to help message 

### PR DESCRIPTION
**Pull request by coolljt0725**
1. The first commit fix the the wrong name of `OLLAMA_LLM_LIBRARY` in `map[string]EnvVar`
2. The second commit add more env vars to help message of `ollama serve --help` to let user know what env var we can use when start the server.

### CODE REVIEW

**Review by jmorganca — APPROVED** [review]

File: cmd/cmd.go [hunk 1]
```diff
@@ -1251,6 +1251,10 @@ func NewCLI() *cobra.Command {
 				envVars["OLLAMA_NOPRUNE"],
 				envVars["OLLAMA_ORIGINS"],
 				envVars["OLLAMA_TMPDIR"],
+				envVars["OLLAMA_FLASH_ATTENTION"],
+				envVars["OLLAMA_LLM_LIBRARY"],
+				envVars["OLLAMA_MAX_VRAM"],
+				envVars["OLLAMA_RUNNERS_DIR"],
```
Comment (jmorganca) [inline cmd/cmd.go:1257]: Let's remove this one as it is primarily for development purposes

Comment (jmorganca) [inline cmd/cmd.go:1257]:
```suggestion
```

File: envconfig/config.go [hunk 2]
```diff
@@ -51,16 +51,16 @@ func AsMap() map[string]EnvVar {
 		"OLLAMA_FLASH_ATTENTION":   {"OLLAMA_FLASH_ATTENTION", FlashAttention, "Enabled flash attention"},
 		"OLLAMA_HOST":              {"OLLAMA_HOST", "", "IP Address for the ollama server (default 127.0.0.1:11434)"},
 		"OLLAMA_KEEP_ALIVE":        {"OLLAMA_KEEP_ALIVE", KeepAlive, "The duration that models stay loaded in memory (default \"5m\")"},
-		"OLLAMA_LLM_LIBRARY":       {"OLLAMA_ORIGINS", LLMLibrary, ""},
+		"OLLAMA_LLM_LIBRARY":       {"OLLAMA_LLM_LIBRARY", LLMLibrary, "Set LLM library to bypass autodetection"},
```
Comment (jmorganca) [inline envconfig/config.go:54]: Thanks for fixing

**Comment by coolljt0725** [comment]
@jmorganca  updated

**Review by jmorganca — APPROVED** [review]

**Pull request merged**